### PR TITLE
chore(V3template): non-sso-tab-js for V3 template

### PR DESCRIPTION
**Pull request by frankqianms**
1. change bicep files for non-sso-tab-js template
2. fix a bug in sso-tab-js template

### CODE REVIEW

File: templates/scenarios/non-sso-tab-js/tabs/infra/azure.parameters.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "resourceBaseName": {
+        "value": "v3sample${{RESOURCE_SUFFIX}}"
```
Comment (xzf0587) [inline templates/scenarios/non-sso-tab-js/tabs/infra/azure.parameters.json:6]: Update the resource base name by "tab${{RESOURCE_SUFFIX}}"

File: templates/scenarios/non-sso-tab-js/.fx/.env.local [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Built-in environment variables
+TEAMSFX_ENV=local
+
+# Generated during provision, you can also add your own variables. If you're adding a secret value, add SECRET_ prefix to the name so Teams Toolkit can handle them properly
```
Comment (hund030) [inline templates/scenarios/non-sso-tab-js/.fx/.env.local:4]: Seems the comments are wrong.

File: templates/scenarios/non-sso-tab-js/tabs/infra/azure.parameters.json (hunk 1 above)
Comment (frankqianms) [reply templates/scenarios/non-sso-tab-js/tabs/infra/azure.parameters.json]: sry for missing here. has been fixed.

File: templates/scenarios/non-sso-tab-js/.fx/.env.local (hunk 2 above)
Comment (frankqianms) [reply templates/scenarios/non-sso-tab-js/.fx/.env.local]: unnecessary comments have been deleted now

Comment (hund030) [reply templates/scenarios/non-sso-tab-js/.fx/.env.local]: I mean they look different from those in sso-tab-js/.fx/.env.local. Is it expected?

Comment (frankqianms) [reply templates/scenarios/non-sso-tab-js/.fx/.env.local]: sorry for mistaking the comments in *.dev and those in *.local. they are aligned now.

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**